lib/isMobilePhone.js

```diff
@@ -80,7 +80,7 @@ var phones = {
   'nl-NL': /^(\+?31|0)6?\d{8}$/,
   'nn-NO': /^(\+?47)?[49]\d{7}$/,
   'pl-PL': /^(\+?48)? ?[5-8]\d ?\d{3} ?\d{2} ?\d{2}$/,
-  'pt-BR': /(?=^(\+?5{2}\-?|0)[1-9]{2}\-?\d{4}\-?\d{4}$)(^(\+?5{2}\-?|0)[1-9]{2}\-?[6-9]{1}\d{3}\-?\d{4}$)|(^(\+?5{2}\-?|0)[1-9]{2}\-?9[6-9]{1}\d{3}\-?\d{4}$)/,
+  'pt-BR': /(?=^(?!020|5520|\+5520|023|5523|\+5523|025|5525|\+5525|026|5526|\+5526|029|5529|\+5529|030|5530|\+5530|036|5536|\+5536|039|5539|\+5539|040|5540|\+5540|050|5550|\+5550|052|5552|\+5552|056|5556|\+5556|057|5557|\+5557|058|5558|\+5558|059|5559|\+5559|060|5560|\+5560|070|5570|\+5570|072|5572|\+5572|076|5576|\+5576|078|5578|\+5578|080|5580|\+5580|090|5590|\+5590)(\+?5{2}\-?|0)[1-9]{2}\-?\d{4}\-?\d{4}$)(^(\+?5{2}\-?|0)[1-9]{2}\-?[6-9]{1}\d{3}\-?\d{4}$)|(^(?!.20|023|025|026|029|030|036|039|040|050|052|056|057|058|059|060|070|072|076|078|080|090)((?!\+5520|5520|\+5523|5523|\+5525|5525|\+5526|5526|\+5529|5529|\+5530|5530|\+5536|5536|\+5539|5539|\+5540|5540|\+5550|5550|\+5552|5552|\+5556|5556|\+5557|5557|\+5558|5558|\+5559|5559|\+5560|5560|\+5570|5570|\+5572|5572|\+5576|5576|\+5578|5578|\+5580|5580|\+5590|5590)\+?5{2}\-?|0)[1-9]{2}\-?9[6-9]{1}\d{3}\-?\d{4}$)/,
```

Comment:
Yes, this should be simplified.

Reply:
Perfect, thanks!

Reply:
Hello! Thanks for your feedback! Please, can you check its ok now in my new commit?